src/bootstrap-public-path.js

```diff
@@ -0,0 +1,4 @@
+/**
+ * Created by welchk on 7/20/16.
+ */
+__webpack_public_path__ = window.location.protocol + "//" + window.location.host + "/";
```

Comment:
This is dynamically sets the public path at runtime allowing proper resolving for loaders. 

Found it in the issue for relative paths in `style-loader`. 
Further explaination: https://github.com/webpack/style-loader/pull/96#issuecomment-228370602

Honestly, I am not a fan of this outside of the fact that it does work. This will allow proper support icon support to fix #125, but it also does so in a way that does not cause other breaking changes as seen in #181 & #205.